tanzu-application-platform/README.md

```diff
@@ -0,0 +1,178 @@
+
+# Deploy to Tanzu Application Platform (TAP)
+
+Animal Rescue supports configuration to build and deploy to Tanzu Application Platform (TAP).  The following instructions describe an end-to-end 
+process for configuring, building, and deploying the Animal Rescue application to a TAP `full` profile cluster.  Using additional configuration, the process can
+be split across multiple cluster profiles (`build, view, and run`).
+
+## Prerequisites
+
+These instructions assume that you have a TAP 1.5.x or greater `full` profile cluster up and running with the following packages installed and [kubectl](https://kubernetes.io
+docs/tasks/tools/) and the Tanzu CLI installed and configured to access your TAP cluster:
+
+* YTT
```

Comment:
```suggestion
* [ytt](https://carvel.dev/ytt/)
```
Uun-familiar people googling `ytt` may end up on other pages without a hint.

Reply:
Fair point.  Updated in the PR.